certbot_external_auth/plugin.py

```diff
@@ -724,6 +727,26 @@ def _call_handler(self, command, *args, **kwargs):
     # Helper methods & UI
     #
 
+    def _json_sanitize_dict(self, dictionary):
+        for key, val in list(dictionary.items()):
+            # Not highly effecient, would be neater to clean up FIELD_TOKEN.
+            # But if any of the others turn to bytes in the future, this will solve it:
+            if type(key) == bytes:
+                del dictionary[key]
+                key = key.decode('UTF-8')
+                dictionary[key] = val
+
+            if type(val) == bytes:
+                dictionary[key] = val.decode('UTF-8')
+            elif type(val) in (list, tuple):
```

Comment:
I am thinking whether 
a) if we want to handle also lists - maybe make the function recursive totally, e.g., it would handle dictionaries, plain scalar values, lists, tuples, ...

```        
@staticmethod
def _json_sanitize_dict(value):
        if isinstance(value, bytes):
            return value.decode('UTF-8')
        if isinstance(value, types.TupleType):
            return tuple([_json.sanitize(x) for x in value])
        if isinstance(value, types.ListType):
            return [_json.sanitize(x) for x in value]
        if isinstance(value, types.DictionaryType):
            return {key: _json.sanitize(value[key]) for key in value}
        return value
```

b) do not handle lists as we dont need it now